src/neuroconv/datainterfaces/ecephys/openephys/openephys_utils.py

```diff
@@ -0,0 +1,52 @@
+from datetime import datetime
+from pathlib import Path
+from typing import Union
+from warnings import warn
+
+from lxml import etree
+
+from ....utils import FolderPathType
+
+
+def get_session_start_time(element: etree.Element) -> Union[datetime, None]:
```

Comment:
Also these, since as a public function it could invoke namespace collisions with other past functions (from before we started following sklearn style with new additions)

```suggestion
def _get_session_start_time(element: etree.Element) -> Union[datetime, None]:
```

Reply:
Also leaves us free to rename it in the future if we ever want to standardize the structure of similar functions more across formats

Reply:
Sure, thanks for the suggestion.